code/modules/mob/living/living.dm

```diff
@@ -998,7 +998,7 @@
 /mob/living/proc/attempt_harvest(obj/item/I, mob/user)
 	if(user.a_intent == INTENT_HARM && stat == DEAD && butcher_results) //can we butcher it?
 		var/sharpness = is_sharp(I)
-		if(sharpness)
+		if(sharpness && I.force >= 5)
```

Comment:
Please put this in the Changelog or the PR description

Reply:
fuck